### DISCUSSION TITLE
Accessibility (for blind)

### DISCUSSION THREAD

**Discussion by pikurasa** [sugarlabs/musicblocks-v4#27] [Ideas]
Right now, musicblocks is made with sighted people in mind.

However, music is inherently accessible to blind, since it requires only hearing.

Let's create a text-based layer that can be read by screen-readers so that blind can use music blocks.

**Comment by valessiobrito** [comment]
Suggestion for interview  Sam Hartman,  
He is blind, musician (DJ) also a Debian Developer and was once a Debian Project Leader[1].

[1] https://www.debian.org/vote/2019/platforms/hartmans
